content/en-US/api/Serial.md

```diff
@@ -0,0 +1,86 @@
+---
+recipe: api-interface
+title: 'Serial'
+mdn_url: /en-US/docs/Web/API/Serial
+specifications: https://wicg.github.io/serial/#serial-interface
+browser_compatibility: api.Serial
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The `Serial` interface of the Web Serial API provides attributes and methods for
+finding and connecting to serial ports from a web page.
+
+This interface requires a secure context.
+
+## Constructor
+
+None.
+
+## Events
+
+**`Serial.onconnect`**
+
+Called when the port has been connected to the device. This event is only fired
+for ports associated with removable devices such as those connected via USB. The
+target of this event is the `SerialPort` interface that has been added. The user
+must grant the origin permission to access this device during a call to
+`requestDevice()` before this event will be fired.
+
+**`Serial.ondisconnect`**
+
+Called when the port has been connected to the device. This event is only fired
+for ports associated with removable devices such as those connected via USB. The
+target of this event is the `SerialPort` interface that has been removed. The
+user must grant the origin permission to access this device during a call to
+`requestDevice()` before this event will be fired.
```

Comment:
```suggestion
target of this event is the `SerialPort` interface that has been removed. 
```

Reply:
Why do you want to remove the final sentence?

Reply:
I do not remember.